.github/workflows/release.yml

```
@@ -32,10 +32,13 @@ on:
 permissions:
   contents: write
 
+# 同一 tag/PR 的重复运行整体排队，平台内部再通过 needs 链串行写 Release。
+concurrency:
+  group: release-workflow-${{ github.ref }}
+  cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)